### PR TITLE
Web 22 community announcements

### PR DESCRIPTION
**Pull request by andygimma**

### CODE REVIEW

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -233,9 +233,11 @@ type Post {
   attachments(type: String): [Attachment]
   attachmentsTotal: Int
   postMemberships: [PostMembership]
+  postMembershipsTotal: Int
```
Comment (robbiecarlton) [inline api/graphql/schema.graphql]: duplication

File: knexfile.js [hunk 2]
```diff
@@ -32,5 +32,14 @@ module.exports = {
   development: defaults,
   dummy: Object.assign({}, defaults, { seeds: { directory: './seeds/dummy' } }),
   staging: defaults,
-  production: merge({connection: {ssl: true}}, defaults)
+  production: merge({connection: {ssl: true}}, defaults),
+  docker: Object.assign({},
```
Comment (robbiecarlton) [inline knexfile.js]: should this be part of this PR? Not sure what it's doing

File: migrations/20180315140159_add_announcement_to_post.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('posts', t => {
+    t.boolean('announcement').defaultTo(false)
+  })
+};
+
+exports.down = function(knex, Promise) {
```
Comment (robbiecarlton) [inline migrations/20180315140159_add_announcement_to_post.js]: the down function should drop the column ie
```
  return knex.schema.table('posts', t => t.dropColumn('announcement'))

```

File: migrations/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+postgres:
```
Comment (robbiecarlton) [inline migrations/docker-compose.yml]: same question

File: knexfile.js (hunk 2 above)
Comment (andygimma) [reply knexfile.js]: @richchurcher made this change to allow for easier migrations.

Comment (robbiecarlton) [reply knexfile.js]: cool

**Pull request merged**